packages/pages/src/common/src/project/structure.ts

```diff
@@ -17,6 +18,12 @@ export interface ProjectFilepaths {
   hydrationBundleOutputRoot: string;
   /** The folder path where the compiled server bundles should go */
   serverBundleOutputRoot: string;
+  /**
+   * The domain of the site, which also serve as the subfolder name
+   * inside {@link templatesRoot} and {@link sitesConfigRoot}.
+   * This is used for the case of multibrand setup within a single repo.
+   */
+  domain?: string;
```

Comment:
Not sure exactly what the spec said here but I don't think we should use the word "domain" here. "scope" makes more sense to me as you're scoping a build to a subset of templates. What do you all think?

Reply:
Hmm, we are not fully scoped to just the specified subset since we do use other files in root folder (e.g. a file with name exist only in root will be share across all brand folders, or something specified in a domain features.json but doesn't exist in that template subfolder would fallback to root)

Reply:
updated the name to `scope`